src/main/java/wolox/training/models/Book.java

```diff
@@ -118,6 +124,7 @@ public String getPublisher() {
     }
 
     public void setPublisher(String publisher) {
+        Preconditions.checkNotNull(publisher);
```

Comment:
The same as the previous comment.

Reply:
I put the messages and added it like constants.